service/controller/appcatalog/v1/resource/index/desired_test.go

```diff
@@ -0,0 +1,136 @@
+package index
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/micrologger/microloggertest"
+	"github.com/google/go-cmp/cmp"
+	"k8s.io/api/core/v1"
```

Comment:
```suggestion
	corev1 "k8s.io/api/core/v1"
```

Another alias to set. Since we use the k8s packages so often using the naming convention is important.